package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "front-sdk",
+  "version": "0.0.1",
+  "description": "A JavaScript API for Front (https://frontapp.com)",
+  "homepage": "https://github.com/resin-io-modules/front-sdk#readme",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
```

Comment:
If you point this at `src/index.ts` then there's no need to generate a `.d.ts` file